01-JS-DOM-APIs/Exercises1_2/js/main.js

```diff
@@ -0,0 +1,12 @@
+window.onload = function(a)
+{
+    hidden();
+    document.getElementById("btn").addEventListener("click", function showAlert(){
+        alert("You push!");
+    });
+    
+};
+
+function hidden(){
```

Comment:
The effect isn't working as expected. The animation should be gradual. Try using CSS.